packages/Babylonian-Core.package/BPMethodExample.class/class/bpExamplePragmaSelectors.st

```diff
@@ -1,8 +1,8 @@
 constants
 bpExamplePragmaSelectors
-
+	
 	"printIt: [(0 to: 6) gather: [:i |
-		#(exampleNamed: exampleNamed:active: exampleNamed:active:self: exampleNamed:self:) collect: [:prefix |
+		#(exampleNamed:expectedResult: exampleNamed:expectedResult:active: exampleNamed:expectedResult:active:self: exampleNamed:expectedResult:self:) collect: [:prefix |
```

Comment:
This makes expectedResult a mandatory keyword for all examples. It would be nicer to keep it optional.

Reply:
ah i see. ok

Reply:
this will become a hassle with the active keyword tho :D